Jellyfin.ApiClient/Net/NetworkConnection.cs

```diff
@@ -53,7 +52,7 @@ private async Task SendWakeOnLan(string macAddress, IPEndPoint endPoint, Cancell
             const int payloadSize = 102;
 
             var macBytes = PhysicalAddress.Parse(macAddress).GetAddressBytes();
-            _logger.Debug(string.Format("Sending magic packet to {0}", macAddress));
+            _logger.LogDebug(string.Format("Sending magic packet to {0}", macAddress));
```

Comment:
This entire class was only used for WoL so it was removed from the Java client.

Reply:
Removed everything that explicitly mentioned WoL, network detection stuff seems a bit tangled with server detection which I'll be cleaning up in a future PR